commitlintplugins.test.ts

```diff
@@ -152,6 +152,20 @@ test('prefer-slash-over-backslash2', () => {
 });
 
 
+test('reject-hashtag-refs1', () => {
+    let commitMsgWithHashtagRef = "foo: fixes #123";
+    let rejectHashtagRefs1 = runCommitLintOnMsg(commitMsgWithHashtagRef);
+    expect(rejectHashtagRefs1.status).not.toBe(0);
+});
+
+
+test('reject-hashtag-refs2', () => {
+    let commitMsgWithFullUrl = "foo: blah blah" + '\n\n' + 'Fixes someUrl://blah.blah/158';
+    let rejectHashtagRefs1 = runCommitLintOnMsg(commitMsgWithFullUrl);
+    expect(rejectHashtagRefs1.status).toBe(0);
+});
+
+
```

Comment:
@realmarv you're missing a test that has a #123 inside a 3-tick block.

Reply:
Done